common/test/acceptance/tests/lms/test_lms_cohorted_courseware_search.py

```diff
@@ -0,0 +1,245 @@
+"""
+Test courseware search
+"""
+import os
+import json
+
+from ...pages.common.logout import LogoutPage
+from ...pages.studio.overview import CourseOutlinePage
+from ...pages.lms.courseware_search import CoursewareSearchPage
+from ...fixtures.course import XBlockFixtureDesc
+
+from nose.plugins.attrib import attr
+
+from ..studio.base_studio_test import ContainerBase
+
+from ...pages.studio.settings_group_configurations import GroupConfigurationsPage
+from ...pages.studio.auto_auth import AutoAuthPage as StudioAutoAuthPage
+from ...fixtures import LMS_BASE_URL
+from ...pages.studio.component_editor import ComponentVisibilityEditorView
+from ...pages.lms.instructor_dashboard import InstructorDashboardPage
+
+from bok_choy.promise import EmptyPromise
+
+
+@attr('shard_1')
+class CoursewareSearchCohortTest(ContainerBase):
```

Comment:
It seems like we'd really want a normal python unit test for this - see the newly added `cms/djangoapps/contentstore/tests/test_courseware_index.py` file in master - might be possible to add some tests therein, and not worry about needing to do bok choy for this specific test.

bokchoy is necessary to test the indexing => searching flow within the environment, we already have this - it might be easier to have just unit tests for cohort search.

On the other hand these seem fine, but I'd like to have cohort tests in the regular unit tests, so you can choose to keep these or not, but I'd really like unit tests.